proto/launch/tx.proto

```diff
@@ -48,7 +48,9 @@ message MsgEditChain {
   string sourceURL = 4;
   string sourceHash = 5;
   InitialGenesis initialGenesis = 6;
-  bytes metadata = 7;
+  bool setCampaignID = 7;
```

Comment:
the campaign id cannot be `0`. Maybe we don't need the `setCampaignID` flag

Reply:
> the campaign id cannot be 0. Maybe we don't need the setCampaignID flag

It can by design if we set the initial campaign counter to `0`, I would prefer to not rely on this assumption